lib/travis/build/script/julia.rb

```diff
@@ -71,6 +73,11 @@ def script
                 '-e "Pkg.test(\"${JL_PKG}\", coverage=true)"'
             end
           end
+          sh.else do
+            sh.echo '\`src/${JL_PKG}.jl\` not found, repository is not a '\
+              'valid Julia package so the default test script is empty',
+              ansi: :green
```

Comment:
Perhaps another color (`:yellow`) may draw more attention here? (Not sure if this is necessarily an error condition, but it may not be something the user is expecting.)